spec/lib/traject/config/title_spec.rb

```diff
@@ -54,16 +64,35 @@
       result = select_by_id('245nAndp')[field]
       expect(result.first).to include 'humanities'
     end
+    context 'linked fields' do
+      let(:fixture_name) { 'vernacularSearchTests.mrc' }
+      it 'are not included' do
+        expect(select_by_id('2xxVernSearch')[field]).not_to eq ['vern245a',
```

Comment:
With the below expectation `[field]).to eq ` can we get away with out adding this test?